shared/wallets/wallets-and-details/index.js

```diff
@@ -1,20 +1,24 @@
 // @flow
 import * as React from 'react'
-import {Box2} from '../../common-adapters'
+import {Box2, Reloadable} from '../../common-adapters'
 import WalletList from '../wallet-list/container'
 import {globalColors, styleSheetCreate} from '../../styles'
+import {loadAccountsWaitingKey} from '../../constants/wallets'
 
 type Props = {
   children: React.Node,
+  reload: () => void,
 }
 
 const Wallets = (props: Props) => (
-  <Box2 direction="horizontal" fullHeight={true} fullWidth={true}>
-    <Box2 direction="vertical" fullHeight={true} style={styles.walletListContainer}>
-      <WalletList style={{height: '100%'}} />
+  <Reloadable waitingKeys={loadAccountsWaitingKey} onReload={props.reload} reloadOnMount={true}>
```

Comment:
One can still see black bars for other wallet-related RPCs. Not sure how to fix that though -- The set of waiting keys is dependent on the list of wallet accounts.

Reply:
Since we do `thing:${id}` for these often it'd make sense if `Reloadable` could take a prefix to watch out for (not necessary in this PR)

Reply:
Could be a filter function